db/schema.rb

```diff
@@ -28,6 +28,38 @@
     t.index ["resource_type", "resource_id", "action", "identifier"], name: "index_on_resource_action_identifier", unique: true
   end
 
+  create_table "cluster_tags", id: :serial, force: :cascade do |t|
+    t.bigint "tag_id", null: false
+    t.bigint "cluster_id", null: false
+    t.datetime "last_seen_at"
+    t.index ["cluster_id"], name: "index_cluster_tags_on_cluster_id"
+    t.index ["last_seen_at"], name: "index_cluster_tags_on_last_seen_at"
+    t.index ["tag_id", "cluster_id"], name: "index_cluster_tags_on_tag_id_and_cluster_id", unique: true
+  end
+
+  create_table "clusters", force: :cascade do |t|
+    t.bigint "tenant_id", null: false
+    t.bigint "source_id", null: false
+    t.string "source_ref", null: false
+    t.string "uid_ems"
+    t.string "name"
+    t.jsonb "extra"
+    t.datetime "resource_timestamp"
+    t.jsonb "resource_timestamps", default: {}
+    t.datetime "resource_timestamps_max"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.datetime "archived_on"
```

Comment:
@agrare this is `archived_at` right? (maybe we could add this to the schema health specs, making sure there are no `_on` timestamps)

Reply:
Ah right I forgot we fixed that, fixed